euth/contrib/static/js/app.js

```diff
@@ -15,3 +15,23 @@ module.exports = {
   'renderFollow': ReactFollow.renderFollow,
   'renderUserList': ReactUserList.renderUserList
 }
+
+var initilizeWidget = function (project, name, initializer) {
+  if (!initializer) {
+    initializer = name
+    name = project
+    project = 'euth'
+  }
+
+  document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I wanted to avoid a jQuery dependency here. At a quick internet search yielded this as a equivalent vanilla java script option.